.github/workflows/main.yaml

```diff
@@ -0,0 +1,47 @@
+name: Main
+on: [ pull_request ]
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    env:
+      BRANCH: ${{ github.head_ref }}
+      AWS_DEFAULT_REGION: "us-west-2"
+      ALGOLIA_PRIVATE_KEY: ${{ secrets.AGOLIA_PRIVATE_KEY }}      
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    # - name: Login to docker
+    #   uses: docker/login-action@v2
+    #   with:
+    #     username: ${{ secrets.DOCKER_USERNAME }}
+    #     password: ${{ secrets.DOCKER_TOKEN }}
+    - name: Set vars
+      run: |
+        if [ "$BRANCH" = "main" ]; then 
+          BUCKET=preview
+          HOSTNAME=dev-docs.d2iq.com
```

Comment:
this site doesn't build to dev docs anymore.

Reply:
what does it build to?

Reply:
Also this is the what the jenkins job is building to currently..
https://github.com/mesosphere/dcos-docs-site/blob/main/Jenkinsfile#L10

Reply:
the `main` branch should deploy to dev-docs.d2iq.com as well as archive-docs.d2iq.com, it's just that we build different stuff to them based on that hide folders configuration. This may no longer be necessary but its probably just easier to copy over the exact implementation

Reply:
ok thanks for clarifying Russ!